src/app/components/JsonSchemaConfigurator/JsonSchemaConfigurator.tsx

```diff
@@ -46,11 +33,13 @@ export const JsonSchemaConfigurator: FunctionComponent<JsonSchemaConfiguratorPro
         model={configuration}
         onChangeModel={(model: any) => onChange(model, false)}
         onSubmit={(model: any) => onChange(model, true)}
-        // autosave
-        // autosaveDelay={0}
+        className="configurator"
       >
-        <AutoFields />
-
+        <Card isPlain>
+          <CardBody>
+            <AutoFields omitFields={['processors', 'error_handler']} />
```

Comment:
@indraraj @uidoyen what about `data_shape`? shouldn't this be omitted too?

Reply:
@rk3rn3r no because we only want the `error_handler` and `processor` to be present on the separate step/page for camel connectors. `data_shape` will be presented within the `Connector specific` step.